src/Illuminate/Foundation/Vite.php

```diff
@@ -655,6 +655,28 @@ public function asset($asset, $buildDirectory = null)
         return $this->assetPath($buildDirectory.'/'.$chunk['file']);
     }
 
+    /**
+     * Get the content for an asset.
+     *
+     * @param  string  $asset
+     * @param  string|null  $buildDirectory
+     * @return string
+     */
+    public function content($asset, $buildDirectory = null)
+    {
+        $buildDirectory ??= $this->buildDirectory;
+
+        $chunk = $this->chunk($this->manifest($buildDirectory), $asset);
+
+        $path = public_path($buildDirectory.'/'.$chunk['file']);
+
+        if (! is_file($path) || ! file_exists($path)) {
+            return '';
```

Comment:
I wonder whether this should throw an exception, similar to what the `chunk` method will do if the file doesn't exist in the manifest.

Reply:
@jessarcher I've added the exception.
Can you have a look if that's what you'd expect?